### ISSUE TITLE
Networkmanager reference is null on SpectatorView

### ISSUE THREAD

**Issue by benzben** [Microsoft/MixedRealityToolkit-Unity#3103]
**Does this affect the legacy HoloToolkit (master) or the Mixed Reality Toolkit (mrtk_release)?**
HoloToolkit

**Describe the bug**
If I drag the SpectatorView.prefab into my scene (I'm following this instruction (https://docs.microsoft.com/en-us/windows/mixed-reality/spectator-view) Unity tells me that Networkmanager reference is null on SpectatorView, without hitting play or trying to build.
Also when trying to build it fails because of this error.

**Unity Editor Version**
2017.3 and 2018.1.0f2

**Additional context**
The component 'NetworkManager' seems to be missing in the prefab:
![grafik](https://user-images.githubusercontent.com/39216624/48188653-91e54780-e33e-11e8-80a1-dbc0d2ebf049.png)
Also I tried to use a FindObjectOfType(...) in the Awake method, but this did not result in success. In the instructions I followed, they have a screenshot where there's clearly an existing component NetworkManager. I downloaded the latest Git repo, so in there this component may be missing? Or am I clearly blind to something obvious?

**Comment by JoseSantano-ms** [comment]
Hello @benzben 

**TL;DR add  your own.** 

The new version of Spectator View doesn't ship with a Network Manager. the reason behind it is that you application has to be Multilense before you add Spectator View, or at least it has to be once you do it. We don't want to enforce people to use our implementation because every application is different. And of top of that we want to make Spectator View easy to add onto an application already built. 

If you look at the example you'll see that there's a very basic `NetworkManager` component. To begin with, something along those lines will do :)
